website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Migrating Legacy Projects
+linkTitle: Migrating Legacy Projects
+weight: 7
+description: This guide walks through an example of migrating a simple memcached-operator which was built by following the [legacy quick-start][quickstart-legacy] to the new Kubebuilder-style layout.
+---
+
+## Overview
+
+The motivations for the new layout are related to bringing more flexibility to users and 
+part of the process to [Integrating Kubebuilder and Operator SDK][integration-doc].
+
+### What was changed
+ 
+The `deploy` directory was replaced with the `config` directory including a new layout of Kubernetes manifests files
+- CRD manifests in `deploy/crds/` are now in `config/crd/bases`
```

Comment:
For this section, please apply the same suggestions as helm. https://github.com/operator-framework/operator-sdk/pull/3595#pullrequestreview-458361167